lib/paginate.rb

```diff
@@ -0,0 +1,4 @@
+module Paginate
```

Comment:
This is a good way to start, you can create a method in this module to change the scope of queries, just like the one from the gem.